shardingsphere-sql-parser/shardingsphere-sql-parser-statement/src/main/java/org/apache/shardingsphere/sql/parser/sql/common/util/TableExtractor.java

```diff
@@ -46,7 +46,7 @@
 import java.util.LinkedList;
 import java.util.Optional;
 
-public final class TableExtractUtils {
+public final class TableExtractor {
```

Comment:
Is the class TableExtractor suitable in util package?
if it is not an util class, please move it to correct package.

Reply:
I think it is just a util class with status, what folder do you think is more appropriate?

Reply:
maybe extractor is a better name

Reply:
Ok